journals/apps/journals/blocks.py

```diff
@@ -16,9 +18,17 @@
 STREAM_DATA_TYPE_FIELD = 'type'
 STREAM_DATA_DOC_FIELD = 'doc'
 
+SPAN_ID_FORMATTER = '{doc_type}-{doc_id}'
+BLOCK_FORMATTER = '{span} {block}'
```

Comment:
This can be changed to 
`BLOCK_FORMATTER = '{block_prefix} {block}'`

Reply:
yes , I will